
beginner_tutorials/testMsg TestMsg

int64 a
int64 b
int64 c
---
int64 sum

